pages/filters/Create/CreateHeader.vue

```diff
@@ -0,0 +1,19 @@
+<template>
```

Comment:
The naming structure of the entire pages folder is still not consistent. Some are uppercase some lowercase some folders have a index.vue some dont.